### DISCUSSION TITLE
How to input a new account

### DISCUSSION THREAD

**Discussion by Hanatt27e** [2fast-team/2fast#78] [Q&A]
Hello!!

I created a new account but the OTP code is now working, I'm still not sure how to link the account I want to get access to the account I created, can you please advise,
Thanks!

**Comment by jp-weber** [comment]
What steps did you take? When you activate two-factor authentication on an account, a QR code is generated:
1. Start 2fast app
2. Click 'Add account' button
3. Click scan button
4. Scan the QR code and go back to the app
5. If the QR code has been read correctly, the account is displayed with the information
6. Click confirm for the new account

The account is now created in the app, in order to establish the link, the generated key that is displayed for the account must be copied and entered in the field for the activated service in the browser/app.
